### PR TITLE
Add the How-To Guide for Multi-Scale Feature Extraction

### PR DESCRIPTION
**Pull request by DexterJZ**
Add the EN&CN versions of the how-to guide for multi-scale feature extraction.

Thank you for your contribution to the MindCV repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs
#702 

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]

**Comment by geniuspatrick** [comment]
1. 放在`docs/en/how_to_guides`文件夹下面
2. 添加一个中文翻译，没有就建一个空白文档，写上`即将到来`
3. 在`mkdocs.yml`里面第19行加上一条

File: docs/en/how_to_guides/feature_extraction.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
+# MindCV Model Feature Extraction
```
Comment (geniuspatrick) [inline docs/en/how_to_guides/feature_extraction.md:1]: Feature Extraction or Pyramidal Feature Extraction or Multi-scale Feature Extraction

File: docs/zh/how_to_guides/feature_extraction.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
+# MindCV模型特征抽取
```
Comment (geniuspatrick) [inline docs/zh/how_to_guides/feature_extraction.md:1]: 特征抽取；金字塔特征抽取；多尺度特征提取

File: mkdocs.yml [hunk 3]
```diff
@@ -16,6 +16,7 @@ nav:
       - Deployment: tutorials/deployment.md
   - How-To Guides:  # Problem Oriented
       - Write A New Model: how_to_guides/write_a_new_model.md
+      - MindCV Model Feature Extraction: how_to_guides/feature_extraction.md
```
Comment (geniuspatrick) [inline mkdocs.yml:19]: 同上，选一个，保持一致即可

**Review by geniuspatrick — APPROVED** [review]

File: docs/zh/how_to_guides/feature_extraction.md (hunk 2 above)
Comment (yanglixiaoshen) [reply docs/zh/how_to_guides/feature_extraction.md]: 您好，我使用了mindcv提供的多尺度特征提取教程对swintransformer进行处理，发现存在raise NotImplementedError的错误。如果您有空您能帮我看看吗？谢谢

**Pull request merged**